uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-notification.component.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-env amd */
+
+define([
+  'leave-absences/shared/modules/components'
```

Comment:
why shared? this is an entire separate angular app, no? (like my-leave, manager-leave, etc)